Source/Tests/skip_tests.lst

```diff
@@ -1,4 +1,7 @@
 test_DefaultTemplateMeasurement_GetUiDetails_CorrectUiPathUrl.vi
 test_NoServicesRegistered_InvalidRegisterRequests_Error.vi
 test_SingleServiceRegistered_DiscoveryApiRoundTrip_AllNodesSucceed.vi
-test_SingleServiceRegistered_InvalidResolveRequests_Error.vi
\ No newline at end of file
+test_SingleServiceRegistered_InvalidResolveRequests_Error.vi
```

Comment:
I have included these tests to skip to ensure LabVIEW doesn't hang while running tests. We are currently working on resolving the LabVIEW hang issue, once resolved these tests can be removed from here.